configs/codon.config

```diff
@@ -0,0 +1,56 @@
+workDir = '/hps/nobackup/rdf/metagenomics/service-team/users/louison/nextflow/tmpwork/'
+
+params {
+    catDB = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/cat/CAT_prepare_20210107/2021-01-07_CAT_database/"
+    diamondDB = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/cat/CAT_prepare_20210107/Diamond_2.0.6/diamond"
+    taxonomyDB = "/hps/nobackup/rdf/metagenomics/service-team/ref-dbs/cat/CAT_prepare_20210107/2021-01-07_taxonomy/"
+    tiaraVersion = "1.0.3"
+    dmcVersion = "1.0.3"
```

Comment:
Add the static models paths here as well once you have them (can be a temp dir in your hps location for now)